docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/movement/col2im-15.rst

```diff
@@ -0,0 +1,198 @@
+
+
+where ``d`` is over all spatial dimensions.
+
+The ``input`` blocks are being moved into the ``output`` tensor of shape ``(N, C, output_size[0], output_size[1])`` by combining the values contained in blocks.
+
+Non-batched inputs are also supported, in which case the ``input`` has shape ``(C * Product(kernel_size), L)`` and the output has shape ``(C, output_size[0], output_size[1])``.
+
+**Attributes**:
+
+* *strides*
+
+  * **Description**: stride in the sliding blocks in the input spatial dimensions.
+  * **Range of values**: 1D tensor of non-negative integer numbers
+  * **Type**: *T_IND*
+  * **Default value**: [1, 1]
+  * **Required**: *no*
+
+* *dilations*
+
+  * **Description**: controls local stride of the elements.
+  * **Range of values**: 1D tensor of non-negative integer numbers
+  * **Type**: *T_IND*
+  * **Default value**: [1, 1]
+  * **Required**: *no*
+
+* *pads_begin*
+
+  * **Description**: *pads_begin* is a number of zero-value pixels to add to the beginning along each axis. For example, *pads_begin* equal "1,2" means adding 1 pixel to the top of the input and 2 to the left of the input.
+  * **Range of values**: 1D tensor of non-negative integer numbers
+  * **Type**: *T_IND*
+  * **Default value**: [0, 0]
+  * **Required**: *no*
+
+* *pads_end*
+
+  * **Description**: *pads_end* is a number of zero-value pixels to add to the ending along each axis. For example, *pads_end* equal "1,2" means adding 1 pixel to the bottom of the input and 2 to the right of the input.
+  * **Range of values**: 1D tensor of non-negative integer numbers
+  * **Type**: *T_IND*
+  * **Default value**: [0, 0]
+  * **Required**: *no*
+
+**Inputs**
+
+* **1**: *data*
+
+  * **Description**: A batched 3D tensor of type *T* and shape ``(N, C * Product(kernel_size), L)`` or an unbatched 2D tensor of type *T* and shape ``(C * Product(kernel_size), L)``. **Required.**
+  * **Range of values**: 1D tensor of non-negative integer numbers
+  * **Type**: *T*
+
+* **2**: *output_size*
+
+  * **Description**: controls the shape of the spatial dimensions of the output image. **Required.**
+  * **Range of values**: batched 3D or unbatched 2D tensor of non-negative integer numbers
+  * **Type**: *T_IND*
+
+* **3**: *kernel_size*
+
+  * **Description**: size of the sliding blocks. **Required.**
+  * **Range of values**: 1D tensor of non-negative integer numbers
+  * **Type**: *T_IND*
+
+**Outputs**
+
+* **1**: The output tensor the output image of type *T* and shape:
+
+  * ``(N, C, output_size[0], output_size[1])`` in case of batched input,
+  * ``(C, output_size[0], output_size[1])`` in case of non-batched input.
+
+**Types**
+
+* *T*: any supported data type.
+* *T_IND*: ``int64`` or ``int32``.
+
+**Examples**
+
+All examples assume ``C = 3``.
+
+*Example 1: default optional Parameters*
+
+.. code-block:: xml
+   :force:
+
+    <layer ... type="Col2Im" ... >
+        <data output_size="16,16" kernel_size="2,2"/>
```

Comment:
When `output_size` and `kernel_size` moved to the "Inputs" sections, then they should be provided as <input> in the IR examples.